travis-ci/vmtest/run.sh

```diff
@@ -378,10 +378,17 @@ LIBBPF_PATH="${REPO_ROOT}" \
 	VMTEST_ROOT="${VMTEST_ROOT}" \
 	VMLINUX_BTF=${vmlinux} ${VMTEST_ROOT}/build_selftests.sh
 
+declare -A tests_results
```

Comment:
nit: test_results (one result per test, so in English should be "test results")

Reply:
But there may be several tests per results, depending on how we consider things (technically, one result per group of test), and that probably got me confused. `test_results` does sound better, I'll adjust.